packages/devextreme/js/__internal/ui/button/wrapper.ts

```diff
@@ -1,13 +1,16 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
 /* eslint-disable @typescript-eslint/no-unsafe-member-access */
 // eslint-disable-next-line import/named
-import { dxElementWrapper } from '../../core/renderer';
-import ValidationEngine from '../../ui/validation_engine';
-import Component from './common/component';
-import type { Button } from '../ui/button';
-import { Option } from './common/types';
-import { getImageSourceType } from '../../core/utils/icon';
-
-export default class ButtonWrapper extends Component {
+import type { dxElementWrapper } from '@js/core/renderer';
+import { getImageSourceType } from '@js/core/utils/icon';
+import ValidationEngine from '@js/ui/validation_engine';
+import { ComponentWrapper } from '@ts/core/r1/component_wrapper';
+import type { Option } from '@ts/core/r1/types';
```

Comment:
As far as i understand your final goal is to remove R1 at all - does it mean that in future PRs u'll update these types and component_wrapper imports?

Reply:
No, my goal is only to remove generation. They will be removed or refactored + renamed in future, but that's far future